src/org/usfirst/frc4904/sovereignty/TrimmablePIDSensor.java

```diff
@@ -0,0 +1,54 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.InvalidSensorException;
+import org.usfirst.frc4904.standard.custom.sensors.PIDSensor;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.PIDSourceType;
+
+public class TrimmablePIDSensor implements PIDSensor {
+	
+	private double trimIncrement = 0.0;
+	private double trimValue = 0.0;
+	private final PIDSensor sensor;
+	
+	public TrimmablePIDSensor(PIDSensor sensor) {
+		this.sensor = sensor;
+	}
+	
+	public TrimmablePIDSensor(PIDSource source) {
+		this(new PIDSensor.PIDSourceWrapper(source));
+	}
+	
+	@Override
+	public void setPIDSourceType(PIDSourceType pidSource) {
+		sensor.setPIDSourceType(pidSource);
+	}
+	
+	@Override
+	public PIDSourceType getPIDSourceType() {
+		return sensor.getPIDSourceType();
+	}
+	
+	@Override
+	public double pidGet() {
+		return sensor.pidGet() + trimValue;
```

Comment:
There may be issues with switching between kDisplacement and kRate with the same trim value.